src/main/antlr4/imports/mysql_idents.g4

```diff
@@ -56,4 +59,4 @@ fragment BACKTICK: '`';
 QUOTED_IDENT: BACKTICK (('``') | ~('`'))+ BACKTICK;
 
 fragment DIGIT: [0-9];
-WS  :   [ \t\n\r]+ -> skip ;
+WS  :   [ \t\n\r]+ -> channel(HIDDEN) ;
```

Comment:
yeah it does matter, because the rewriter is processing the entire stream, not just the part it's rewriting.  so if you look at how we rewrite, we add directives like "delete the tokens 5 through 12.  add __MAXWELL__ at position 5.  Now give me the entire sql document again."  And the entire SQL document isn't valid unless we preserve whitespace.